ws-nextjs-app/pages/[service]/send/[id]/FormContext/index.tsx

```diff
@@ -137,23 +142,28 @@ export const FormContextProvider = ({
       }
       formData.append(key, fieldValue as string);
     });
-
     try {
       const url = `https://www.bbc.com/ugc/send/${id}?said=${uuid()}`;
 
       const req = new XMLHttpRequest();
       req.responseType = 'json';
       req.open('POST', url, true);
 
+      req.upload.onloadstart = () => {
+        _setScreen('uploading');
```

Comment:
We can remove the `_` at the start of this setter. I just added it originally as its the convention when a variable isn't being used yet.